static/app/bootstrap/legacyTwitterBootstrap.tsx

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Is this coming from bootstrap's source code?

Reply:
If so, it might be worth referencing which commit it's adapted from.

Reply:
No I just re-implemeted it

Reply:
I mean I read the bootstrap code just to make sure I didn't miss anything, but it is pretty different implementation wise, since the bootstrap stuff is all very jquery dependant

Reply:
Ah I see.

Reply:
would any of our frontend tests cover usage of these functions?

Reply:
Looks like we don't have any specific acceptance tests, but I'm really not too worried about them. This functionality is pretty trivial and hasn't changed for a _long_ time.

Reply:
kk did a quick sanity check on safari/chrome/firefox, lgtm 👍🏼